students/kate/session04/mailroom2.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python3
+
+"""
+Mailroom Exercise, session 4
+using dictionaries
+"""
+import math
+
+# define global donor dict and $
+donors_dict = {
+    'Hermione Granger': [43.01, 13.17],
+    'Molly Weasley':87.03,
```

Comment:
the others are lists of donations, this is a single value -- hasn't that caused problems?

it should be a list with a single entry --then you don't need any special case code.